src/main/java/micdoodle8/mods/galacticraft/core/command/CommandGCSpaceProjectTeamAdd.java

```diff
@@ -0,0 +1,96 @@
+package micdoodle8.mods.galacticraft.core.command;
+
+import java.util.List;
+import micdoodle8.mods.galacticraft.api.spaceprojects.ISpaceProject;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.EnumChatFormatting;
+
+public class CommandGCSpaceProjectTeamAdd extends CommandBase implements ISpaceProject {
+
+    @Override
+    public String getCommandName() {
+        return "spinvite";
+    }
+
+    @Override
+    public int getRequiredPermissionLevel() {
+        return 0;
+    }
+
+    @Override
+    public boolean canCommandSenderUseCommand(ICommandSender par1ICommandSender) {
+        return true;
+    }
+
+    @Override
+    public String getCommandUsage(ICommandSender var1) {
+        return "/" + this.getCommandName() + " [User1 Joining User2] [User2's Name]";
```

Comment:
By convention are required parameters in pointy brackets (`<>`) and optional parameters in square brackets (`[]`)

Reply:
didn't know. will change it